### ISSUE TITLE
url before the anchor

### ISSUE THREAD

**Issue by maieul** [goblindegook/littlefoot#24]
Some CMS add an url before the anchor for the footnote.

That make a markup like this

```
<!DOCTYPE html>
<html>
<head>
<link rel="stylesheet" href="https://unpkg.com/littlefoot/dist/littlefoot.css">
</head>
<body>
<!-- Links -->
<p>
  <sup id="fnref:1">
    <a href="index.html#fn:1" rel="footnote">1</a>
  </sup>
</p>

<!-- Footnote List -->
<div class="footnotes">
  <ol>
    <li class="footnote" id="fn:1">
      <p>footnote. <a href="#fnref:1" title="return to article"> ↩</a></p>
      <p></p>
    </li>
  </ol>
</div>
<script src="https://unpkg.com/littlefoot/dist/littlefoot.min.js" type="application/javascript"></script>
<script type="application/javascript">
  littlefoot.default()
</script>
</body>
</html>
```

Supposing the file is called "index.html". While bigfoot was able to manage this case, that is not true for littlefoot.

**Comment by goblindegook** [comment]
Thanks for reporting this, it seems like a bug, will investigate.

**Comment by goblindegook** [comment]
Fixed in 3.0.0.
